libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/Route.kt

```diff
@@ -0,0 +1,12 @@
+package com.mapbox.navigation.base.route.model
+
+data class Route(
+    val routeIndex: String?,
+    val distance: Double,
```

Comment:
Should we make this nullable until this https://github.com/mapbox/mapbox-java/issues/1087 gets fixed?

Reply:
I think if we already got guaranty that it's non-null, we could use it

Reply:
"Problem" is `!!` here https://github.com/mapbox/mapbox-navigation-android/blob/8c70c43183ddc2c2f3b7d3c96c8729badb2d0464/libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/extension/Mappers.kt#L9 as currently could be nullable.